packages-common/pluggable-widgets-tools/configs/webpack.config.dev.js

```diff
@@ -14,13 +14,15 @@ const devConfig = {
     mode: "development",
```

Comment:
🎄 How about the merging of `dev` and `prod` files into one as we discussed? I know it is not needed now, but might allow us to organize code better.

Reply:
People unfortunately is extends webpack.dev.config and prod when they use custom webpacks.. so for now we cant, unless we keep those configs even not using